libs/components/config/src/lib/params.spec.ts

```diff
@@ -277,4 +277,31 @@ describe('SkyAppRuntimeConfigParams', () => {
 
     expect(params.getAllKeys()).toEqual([]);
   });
+
+  it("should exclude certain parameters from being added to a link's querystring", () => {
+    const params: SkyAppRuntimeConfigParams = new SkyAppRuntimeConfigParams(
```

Comment:
TypeScript can infer the type here since you're using a constructor, so you don't need the extra type declaration.

Reply:
The perils of copy/paste. Done.